src/mixins/filing-template-mixin.ts

```diff
@@ -357,17 +360,25 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
         isFutureEffective: this.getEffectiveDateTime.isFutureEffective
       },
       business: {
-        legalType: this.getEntityType,
-        identifier: this.getTempId
+        identifier: this.getTempId,
+        legalType: this.getEntityType
       },
       continuationIn: {
-        existingBusinessInfo: this.getContinuationInBusinessInfo,
-        continuationAuthorization: {},
-        nameRequest: {
-          legalType: this.getEntityType
+        foreignJurisdiction: {
+          country: this.getExistingBusinessInfo?.homeJurisdiction?.country,
+          region: this.getExistingBusinessInfo?.homeJurisdiction?.region || undefined,
+          legalName: this.getExistingBusinessInfo?.legalName,
+          identifier: this.getExistingBusinessInfo?.identifier,
+          incorporationDate: this.getExistingBusinessInfo?.incorporationDate,
+          taxId: this.getExistingBusinessInfo?.taxId || undefined,
+          affidavitFileKey: this.getExistingBusinessInfo?.affidavitFileKey
+        },
+        authorization: {
+          files: [],
+          authorityName: null,
+          date: null,
+          expiryDate: null || undefined
```

Comment:
Some of the above may need to change for the manual/MRAS-entered data. Specifically, if we don't have some of the above data, does the JSON get saved correctly and match the schema?

Reply:
I'm concerned about what the JSON will look like if the user manually entered the existing business info, since not all fields will have values. This code, or the parse code, may need tweaking.

Reply:
Got it. Thanks